doc/notes/post-workshop-todo.md

```diff
@@ -41,6 +81,10 @@ _Should this be in the admin space, or the outcomes space?_
 
 * Overall impressions
 * Changes we'd want to make
+    * Symbology exercise: deliverable in addition to qml files? Screenshot? Discussion post?
+    * More time for exercises, especially on day 1. Maybe fewer topics covered overall?
+    * Should be more clear on file names/structure for git repos. Initially hard to tell which notebooks correspond to each exercise.
+    * Archive workshop materials? E.g., Everything in qgis-data?
```

Comment:
Good idea. Too late already I think for our workshop, but hopefully that thought benefits someone else in the future :)